seminar2/myproject/shop/management/commands/create_order.py

```diff
@@ -0,0 +1,21 @@
+from django.core.management.base import BaseCommand
+from shop.models import Product, Client, Order
+from random import choices, choice
+
+class Command(BaseCommand):
+    help = "Create Order"
+
+    def handle(self, *args, **kwargs):
+        clients = Client.objects.all()
+        products = Product.objects.all()
+        for i in range(1,11):
+            order_product = choices(products, k=i)
+            order_price = 0
+            for product in order_product:
+                order_price += product.price
+            order = Order.objects.create(client = choice(clients),
+                                         total_price = order_price)
+            for product in order_product:
+                order.products.add(product)
+            order.save()
```

Comment:
А вы тестировали эту команду? Просто, насколько я знаю, если не сохранить сперва объект заказа, то в него нельзя будет добавить продукты
```suggestion
            order = Order.objects.create(client = choice(clients),
                                         total_price = order_price)
            order.save()
            for product in order_product:
                order.products.add(product)
            order.save()
```